### PR TITLE
Prevent doubled usage information output

### LINKED ISSUE #809: `now secrets` with no subcommand shows help text twice
Running `now secrets` with no subcommand shows the help text twice:

```console
$ now --version
8.0.2

$ now secrets

  𝚫 now secrets <ls | add | rename | rm> <secret>

  Options:

    -h, --help              Output usage information
    -b, --base64            Treat value as base64-encoded
    -c FILE, --config=FILE  Config file
    -d, --debug             Debug mode [off]
    -t TOKEN, --token=TOKEN Login token

  Examples:

  – Lists all your secrets:

    $ now secrets ls

  – Adds a new secret:

    $ now secrets add my-secret "my value"

    – Once added, a secret's value can't be retrieved in plaintext anymore
    – If the secret's value is more than one word, wrap it in quotes
    – Actually, when in doubt, wrap your value in quotes

  – Exposes a secret as an env variable:

    $ now -e MY_SECRET=@my-secret

    Notice the `@` symbol which makes the value a secret reference.

  – Renames a secret:

    $ now secrets rename my-secret my-renamed-secret

  – Removes a secret:

    $ now secrets rm my-secret

> Error! Please specify a valid subcommand: ls | add | rename | rm

  𝚫 now secrets <ls | add | rename | rm> <secret>

  Options:

    -h, --help              Output usage information
    -b, --base64            Treat value as base64-encoded
    -c FILE, --config=FILE  Config file
    -d, --debug             Debug mode [off]
    -t TOKEN, --token=TOKEN Login token

  Examples:

  – Lists all your secrets:

    $ now secrets ls

  – Adds a new secret:

    $ now secrets add my-secret "my value"

    – Once added, a secret's value can't be retrieved in plaintext anymore
    – If the secret's value is more than one word, wrap it in quotes
    – Actually, when in doubt, wrap your value in quotes

  – Exposes a secret as an env variable:

    $ now -e MY_SECRET=@my-secret

    Notice the `@` symbol which makes the value a secret reference.

  – Renames a secret:

    $ now secrets rename my-secret my-renamed-secret

  – Removes a secret:

    $ now secrets rm my-secret
```

### PR DESCRIPTION
**Pull request by leo**
This closes #809.

### CODE REVIEW

**Comment by Qix-** [comment]
Can we do this differently? Can we instead make `exit` an async function that never resolves and then `await exit(0)`? We have the `exit()` utility for a reason and IMO it should be sufficient in all use cases.

It shouldn't break any other normal uses of it.

**Review by Qix- — CHANGES_REQUESTED** [review]
See above comments.

**Comment by leo** [comment]
@Qix- is correct: https://github.com/nodejs/node/issues/6456

File: src/util/exit.js [hunk 1]
```diff
@@ -1,8 +1,7 @@
-module.exports = code => {
+module.exports = code => new Promise(() => {
```
Comment (Qix-) [inline src/util/exit.js:1]: Lol surprised this didn't trigger a linter error.

cc @sindresorhus - `unicorn/` plugin doesn't do this? O.o

**Review by Qix- — APPROVED** [review]
LGTM, thanks @leo

File: src/util/exit.js (hunk 1 above)
Comment (Qix-) [reply src/util/exit.js:1]: Oh nevermind, we don't use XO in the cli.

**Pull request merged**